modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/AbstractPageMemoryTableStorage.java

```diff
@@ -39,8 +40,8 @@
  * Abstract table storage implementation based on {@link PageMemory}.
  */
 // TODO: IGNITE-16642 Support indexes.
-public abstract class AbstractPageMemoryTableStorage implements TableStorage {
-    protected final AbstractPageMemoryDataRegion dataRegion;
+public abstract class AbstractPageMemoryTableStorage<T extends DataRegion<? extends PageMemory>> implements TableStorage {
+    protected final T dataRegion;
```

Comment:
I can see that `dataRegion` is only stored here for the `createMvPartitionStorage` method which is `TestOnly`. Can we make `createMvPartitionStorage` abstract? This will allow us to:
1. Remove the protected field
2. Remove the generic parameter from this class
3. Remove the cast to `VolatilePageMemoryDataRegion` in `createMvPartitionStorage` implementation

Reply:
fix it